tema1sapt2/tema1sapt2/Program.cs

```diff
@@ -0,0 +1,216 @@
+﻿//Exercise 1<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+Console.Write("Introduceti primul numar" + "  ");
+int num1 = Convert.ToInt32(Console.ReadLine());
```

Comment:
Let's get used to use English only, please. I know I use sometimes Romanian as well but it's not a best practice, I do it out of dispair because I get to use a lot of variables. 😄